### PR TITLE
Products

### PR DESCRIPTION
**Pull request by uday-nv-work**
Added products page and linked it from homepage

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
code is clean . every thing looks good to me

**Review by sumitk-singh — APPROVED** [review]
code looks clean, good to merge

File: css/component.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+@import url("/css/style.css");
+@import url("/components/Products/products.css");
+:root {
+	--COLOR-PRIMARY: #01ccff;
+	--COLOR-SECONDARY: #27e4b5;
+	--COLOR-SOCIAL-ICON: #308ea5;
+	--PRIMARY-HOVER: #7a9ea7;
+	--NOTIFICATION-COLOR: #7cf916;
+	--offgreen: #bacfc0;
+	--darkerofseconadary: #2195d8;
```
Comment (zoyatanweer) [inline css/component.css:11]: two suggestions:

- don't keep the color name as offgreen. What will happen is, if suppose I try to change that color to yellow, then I cannot keep the name as offgreen. So try to keep the names as logical and clear to user as possible.
- seconadary spelling is wrong. Kindly correct it.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	right: 0;
+	z-index: 2;
+	color: var(--off-green);
+	vertical-align: middle;
+}
+.lt {
+	top: 120%;
+	left: 0;
+	z-index: 2;
+	color: aliceblue;
```
Comment (zoyatanweer) [inline css/style.css:70]: use variables here for color

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+	vertical-align: middle;
+}
+.card-change:hover {
+	transform: scale(1.5);
+}
+.button {
+	color: var(--COLOR-BLACK);
```
Comment (zoyatanweer) [inline css/style.css:77]: try to change the color name variable

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+}
+.parent-positioning img {
+	height: 20rem;
+	width: auto;
+}
+.card-margin {
+	margin: 8px 0 0 8px;
+}
+.icon-overlay {
+	right: 20px;
```
Comment (zoyatanweer) [inline css/style.css:125]: write in rem

File: css/component.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+	--BOX-SHADOW: #af9898;
+	--BOX-SHADOW-LIGHTER: #4e4c25;
+	--ALERT-PRIMARY-COLOR: #2563eb;
+	--ALERT-SUCCESS-COLOR: #0de01f;
+	--ALERT-ERROR-COLOR: #e72222;
+	--ALERT-WARNING-COLOR: #f8cb03;
+	--COLOR-WHITE: #ffffff;
```
Comment (zoyatanweer) [inline css/component.css:18]: same, change here too

**Review by zoyatanweer — APPROVED** [review]
good to merge with suggested changes.

**Pull request merged**